src/components/layout/Background.tsx

```diff
@@ -0,0 +1,19 @@
+import { Global } from '@mantine/core';
+import { useConfig } from '../../tools/state';
+
+export function Background() {
+  const { config } = useConfig();
+
+  return (
+    <Global
+      styles={{
+        body: {
+          backgroundImage: `url('${config.settings.background}')` || '',
+          backgroundPosition: 'center',
+          backgroundSize: 'cover',
+          backgroundRepeat: 'no-repeat',
+        },
```

Comment:
This fixes the background scaling issue.
```suggestion
        body: {
          minHeight: '100vh',
          backgroundImage: `url('${config.settings.background}')` || '',
          backgroundPosition: 'center center',
          backgroundSize: 'cover',
          backgroundRepeat: 'no-repeat',
        },
```

Reply:
I tried the same `minHeight: '100vh'` fix that you submitted, and I didn't notice any changes. When I set `backgroundPosition: top` it seemed to fix it. Glad we could figure it out, though!